docs/test/deploy-from-settings.md

```diff
@@ -0,0 +1,47 @@
+# Deploy from settings
+
+## Prerequisites
+Open deploy dialog (+) and select "Specify app details below"
+
+## Expected result at execution
+* In positive case the "Deploy" button must lead to the creation of a running application
+* Validation errors affecting particular fields must be displayed beneath these fields. In case the deployment  fails due to an exception, then a pop dialog is shown
+
+## Specific factors
+
+|Factor                      |1                 |2                                 |3               |4               |5                        |6                    |7               |Comment|
+|----------------------------|------------------|----------------------------------|----------------|----------------|-------------------------|---------------------|----------------|-------|
+|App name                    |Empty             |Correct value                     |Max-length      |Validation error|Same name app exists     |                     |                |       |
```

Comment:
App name has more complexity:

1. The field is used to create a pod, RC, SVC and label. The pod has an auto-generated suffix, but SVC and RC get the name of this field. Existence should also check that part e.g. service exists, but RC not.. Think about if there are more factors
2. The app name is used as a label (think about if this could create some factors) 
3. used in the help text which does have some impact on layouting - I would add this as a comment